server/db/migrations/mysql/000001_create_calls_channels.up.sql

```diff
@@ -0,0 +1,33 @@
+CREATE TABLE IF NOT EXISTS CallsChannels (
```

Comment:
Yeah, not too opinionated here. The only weird thing when it comes to table/column names would be case sensitivity which I understand MySQL may impose. So if anything if we could move to lower case everywhere it may actually simplify a bit.

Reply:
I agree, I think we can use lowercase. In fact IIRC other plugins were doing so.